packages/default/scss/dialog/_layout.scss

```diff
@@ -51,54 +51,21 @@
 
     // Buttons
     .k-dialog-buttongroup {
-        border-width: 1px 0 0;
+        border-width: 0;
```

Comment:
Not sure if we must have top border 1px by default and remove it where we don't need it, or have 0 and add it, where we need it.

@mvgmvg @gyoshev any thoughts on this?

Reply:
IMO we should have a variable with a default value that matches the Default theme, and override it before including the layout file in derived themes, and not include border sizing in the `_theme` file.

Reply:
I will amend the commit